qutebrowser/browser/history.py

```diff
@@ -99,8 +99,7 @@ def __getitem__(self, key):
         return self._new_history[key]
 
     def __iter__(self):
-        return itertools.chain(self._old_urls.values(),
-                               iter(self._new_history))
+        return self._old_urls.values().__iter__()
```

Comment:
Please do `iter(self._old_urls.values())` instead - I think it does the same in this case, but it's usually discouraged to call magic methods directly.